rclcpp/test/rclcpp/executors/test_executors.cpp

```diff
@@ -461,16 +470,16 @@ TYPED_TEST(TestExecutors, spinSome)
   std::thread spinner([&spin_exited, &executor, this]() {
       executor.spin_some(1s);
       executor.remove_node(this->node, true);
-      spin_exited = true;
+      spin_exited.store(true);
     });
 
   // Do some work until sufficient calls to the waitable occur, but keep going until either
   // count becomes too large, spin exits, or the 1 second timeout completes.
   auto start = std::chrono::steady_clock::now();
   while (
     my_waitable->get_count() <= 1 &&
-    !spin_exited &&
-    (std::chrono::steady_clock::now() - start < 1s))
+    !spin_exited.load() &&
+    std::chrono::steady_clock::now() - start < 10s)
```

Comment:
Is this change needed?

Reply:
I don't think so, I can revert it, however, this timeout need not be small, because it's not the purpose of this test to ensure this part happens within a second. If it doesn't it could imply an issue, but again, I don't think that's the point of this test and also it's a fragile way to test it.

Reply:
This was addressed in https://github.com/ros2/rclcpp/pull/2460.